tests/trainers/test_training_loop.py

```diff
@@ -113,9 +134,9 @@ def test_update_frequency_reporting(self):
         def _on_update_end(report, meter, should_log):
             # the losses here should be the sum of two losses in
             # iteration 0 and iteration 1 (both constitute update 0).
-            # Here iter 1 loss: 0.2684, iter 2 loss: 2.4167
+            # Here iter 1 loss: 0.2599, iter 2 loss: 4.2090
             loss = report.losses["loss"].detach().cpu().item()
-            self.assertAlmostEqual(loss, 2.6852, 4)
+            self.assertAlmostEqual(loss, 4.4688, 4)
```

Comment:
this was necessary because I changed the loss calculation to force gradients to be big.